### DISCUSSION TITLE
Pre rendering a cinematic scene in carla

### DISCUSSION THREAD

**Discussion by evoevo07** [carla-simulator/carla#7335] [General]
Hello everyone, 
is it possible in CARLA, for example, to pre-calculate any sequence and then get it as a video with the maximum graphics settings? So, a cinematic video in first person, instead of free movement, like in a video game. I am asking because of hardware limitations, and my main goal is to create a realistic scene with one rgb cam at the front.
I hope I can express my thoughts reasonably well. 
Thank you very much.

**Comment by Blyron** [comment]
Hello!
Yeah is possible but using the UE4 Editor, for that you would need to compile carla from scratch and then do it like you would do in UE4 Editor in any other project. 

Links that may be of your interest:
How to Quickly create a in-game cinematic: https://www.youtube.com/watch?v=GvTzSlcW9oE&ab_channel=IvicaMilari%C4%87
How to build CARLA on Windows: https://carla.readthedocs.io/en/latest/build_windows/
How to build CARLA on Linux: https://carla.readthedocs.io/en/latest/build_linux/

**Reply by evoevo07** [reply]
Thank you very much for the quick responses. I really appreciate it.
I've already done the build and I'm currently working on the cinematics.

Do you happen to know if there are easy ways to make the camera run along the road? Like, similar to a Carla vehicle, but just a camera.

Best regards and thank you very much

**Reply by Blyron** [reply]
Use a cinematic tools for ue4 and use vehicle meshes instead of blueprints, the issue with that is some vehicles meshes does not have doors. so take care of that.
